Calibration/Tools/interface/EcalRingCalibrationTools.h

```diff
@@ -48,7 +49,7 @@ class EcalRingCalibrationTools
   
   static std::atomic<bool> isInitializedFromGeometry_;
 
-  [[cms::thread_guard("isInitializedFromGeometry_")]]
+  CMS_THREAD_GUARD("isInitializedFromGeometry_")
```

Comment:
@fwyzard maybe, I am not experienced enough with macros, but given the definition of `CMS_THREAD_GUARD`, I would rather translate it to this statement:
```
CMS_THREAD_GUARD(isInitializedFromGeometry_)
```

Reply:
Good point... we are already using both syntaxes e.g. in [[1]] and [[2]].
Let me figure out which one is correct and fix the others...

[1]: https://github.com/cms-sw/cmssw/blob/master/DataFormats/Common/interface/DetSetVectorNew.h#L111
[2]: https://github.com/cms-sw/cmssw/blob/master/DataFormats/PatCandidates/interface/PackedCandidate.h#L727

Reply:
I think the correct syntax should be:
```c++
#define CMS_THREAD_GUARD(_var_) [[cms::thread_guard(#_var_)]]
...
CMS_THREAD_GUARD(isInitializedFromGeometry_)
static short endcapRingIndex_[EEDetId::IX_MAX][EEDetId::IY_MAX];
```

Reply:
The present definition
```c++
#define CMS_THREAD_GUARD(_var_) [[cms::thread_guard("#_var_")]]
```
expands to
```
[[cms::thread_guard("#_var_")]]
```
no matter the argument of the macro - which I doubt was the intended behaviour.